go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/docker/docs
 go 1.23.1
 
 require (
-	github.com/docker/buildx v0.20.1 // indirect
-	github.com/docker/cli v27.5.1+incompatible // indirect
+	github.com/docker/buildx v0.21.0 // indirect
+	github.com/docker/cli v28.0.0-rc.2+incompatible // indirect
```

Comment:
this change is related to https://github.com/docker/buildx/pull/2996#issue-2853784500 but the replace directive avoid vendoring this RC so we are safe here.